.travis/obs.sh

```diff
@@ -29,22 +28,24 @@ ${REPONAME} (${FULLVERS}) UNRELEASED; urgency=medium
  -- Travis CI <travis@travis-ci.org>  ${FULLDATE}
 EOF
 
-cat <<EOF > ~/.oscrc
+mkdir -p ~/.config/osc/
+cat <<EOF > ~/.config/osc/oscrc
 [general]
 apiurl = https://api.opensuse.org
 [https://api.opensuse.org]
 user = ${OBSUSER}
 pass = ${OBSPASS}
+credentials_mgr_class=osc.credentials.PlaintextConfigFileCredentialsManager
 EOF
 
 mkdir -p m4
 cp /usr/local/include/bitlbee/*.h facebook
-osc checkout "home:${OBSUSER}" "${REPONAME}" -o /tmp/obs
+osc checkout "home:jgeboski" "${REPONAME}" -o /tmp/obs
 
 (
     cd /tmp/obs
     rm -f *.{dsc,tar.gz}
-    dpkg-source -I -b "${TRAVIS_BUILD_DIR}"
+    dpkg-source -b "${BUILD_DIR}"
```

Comment:
excludes some defaults from the .tar, removed it because debian/changelog and debian/control were not being packaged at some point, let's see if it works with it too :D!